.github/pull_request_template.md

```diff
@@ -26,3 +44,16 @@ What is the _bottom-line_ functionality that needs testing? Describe in pseudo-c
 5. `status` returns ... if `start` failed
 6. `start` fails after `start` succeeded or after `status` says CRC is `Running`
 7. ...
+-->
+
+## Contribution Checklist
+- [ ] I have read the [contributing guidelines](https://github.com/crc-org/crc/blob/main/developing.adoc)
+- [ ] My code follows the style guidelines of this project
+- [ ] I Keep It Small and Simple: The smaller the PR is, the easier it is to review and have it merged
+- [ ] I have performed a self-review of my code
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] I tested my code on specified platforms
```

Comment:
Might be nice to mention that not all need to be checked for platforms; but it helps us to assist/focused in getting it all covered.